### PR TITLE
add MacOS support to run local KinD

### PR DESCRIPTION
**Pull request by tvieira**
This PR adds support for MacOS to run e2e/local/kind scripts (prereq, setup kind and setup test).

### CODE REVIEW

**Comment by tvieira** [comment]
@icygalz you worked on the Debian script for this area, would you mind to take a look this addition?

File: tests/e2e/local/kind/install_prereqs_macos.sh [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+and disk space (these are estimates for a single node kind cluster) dedicated to the
+virtual machine (VM) running the Docker engine otherwise the Kubernetes cluster might
+fail to start up. More info check User Guide: https://kind.sigs.k8s.io/
+"""
+
+install_kubectl
+
+function check_and_install_golang() {
+	echo "Checking Golang is installed..."
+    if ! go help > /dev/null; then
```
Comment (gargnupur) [inline tests/e2e/local/kind/install_prereqs_macos.sh]: nit: indentation looks off

**Comment by gargnupur** [comment]
/lgtm 
looks good to me, but lets wait for John's approval who knows about Kind more

File: tests/e2e/local/kind/install_prereqs_macos.sh (hunk 1 above)
Comment (tvieira) [reply tests/e2e/local/kind/install_prereqs_macos.sh]: Thanks for reviewing @gargnupur. There was a mix between spaces and tab. I got that changed to only spaces.

**Pull request merged**